x/pairing/keeper/fixation_test.go

```diff
@@ -51,7 +51,9 @@ func TestEpochPaymentDeletionWithMemoryShortening(t *testing.T) {
 	// check that both payments were deleted
 	ts.AdvanceEpochs(epochsToSave)
 
-	res, err := ts.QueryPairingListEpochPayments()
+	res, err := ts.QueryProjectDeveloper(clientAcct.Addr.String())
 	require.NoError(t, err)
-	require.Equal(t, 0, len(res.EpochPayments))
+	res2, err := ts.QueryPairingProviderEpochCu(providerAddr, res.Project.Index, ts.spec.Index)
```

Comment:
can we also check the unique session and provider cu were deleted?

Reply:
done. see https://github.com/lavanet/lava/commit/faea0af32417392c127ef850130b024ae9a90c4e